lib/commands/connect/regions.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const cli = require('heroku-cli-util');
+
+const prodRegions = ['us', 'eu', 'tokyo', 'virginia', 'oregon', 'frankfurt', 'sydney'];
```

Comment:
Our goal is to reach dogwood parity by the end of Q2 and then as @lambacck mentioned front run new regions.  If we need to use a static list for now and revisit later that is ok too.